config/owasp/suppressions.xml

```diff
@@ -13,4 +13,10 @@
     <notes><![CDATA[Awaiting new jackson-core dependency update]]></notes>
     <cve>CVE-2020-36518</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[Awaiting new spring framework update]]></notes>
```

Comment:
Could you please add which package it is? Like https://github.com/hmcts/sscs-hearings-api/blob/5dcf606d4cfa44e07c4f33c1677e3ae7fa93858f/config/owasp/suppressions.xml